dev/build_release.py

```diff
@@ -371,11 +371,24 @@ def __gcb_build(cls, name, gradle_root, gcb_service_account, gcb_project):
     if os.path.isdir(gradle_cache):
       # Tell rmtree to delete the directory even if it's non-empty.
       shutil.rmtree(gradle_cache)
-    cmds = [
-      ('gcloud container builds submit --log-http'
-       ' --account={account} --project={project} --config="../{name}-gcb.yml" .'
-       .format(name=name, account=gcb_service_account, project=gcb_project))
-    ]
+    if __self.options.externalize_gcs_upload:
+      tar_filename = name + '-{:%Y%m%d%H%M%S}'.format(datetime.datetime.utcnow())
+      gcs_staging_bucket = __self.options.gcs_staging_bucket or 'gs://spinnaker-build-gcb-staging'
```

Comment:
give the option a default and scrap the or

Reply:
dont you need to add --gce_staging_bucket and --externalize_gcs_upload  to the argparser?

Reply:
Done.